Makefile

```diff
@@ -58,10 +58,16 @@ test:
 
 load-fixtures:
 	docker-compose up -d db
-	echo "Wait for db to be ready..."
+	echo "Wait for database to be ready..."
 	sleep 5
-	docker-compose exec db psql "postgres://postgres:toor@db:5432/bcda?sslmode=disable" -f /var/db/fixtures.sql
-	docker-compose exec db psql "postgres://postgres:toor@db:5432/bcda?sslmode=disable" -f /var/db/synthetic_cclf_files_beneficiaries.sql
+	docker-compose run db psql "postgres://postgres:toor@db:5432/bcda?sslmode=disable" -f /var/db/fixtures.sql
+
+load-synthetic-cclf-data:
+	docker-compose run api sh -c 'tmp/bcda import-synthetic-cclf-package --acoSize=dev --environment=test'
+	docker-compose run api sh -c 'tmp/bcda import-synthetic-cclf-package --acoSize=small --environment=test'
+	docker-compose run api sh -c 'tmp/bcda import-synthetic-cclf-package --acoSize=medium --environment=test'
+	docker-compose run api sh -c 'tmp/bcda import-synthetic-cclf-package --acoSize=large --environment=test'
+	docker-compose run api sh -c 'tmp/bcda import-synthetic-cclf-package --acoSize=extra-large --environment=test'
```

Comment:
Is this one strictly necessary?  If this is just for the travis test environment I think maybe it can be skipped.  There is no harm in leaving it, but it might be 30 seconds of processing we don't need to do.

Reply:
Good question. I was trying to replicate the behavior of the insert script we had before, which included all the sizes. Not sure that we really need it though. @msnook , @tbellj - thoughts?

Reply:
@embh Does it affect performance of test execution?  If not, I don't think it hurts to keep it in.

Reply:
I believe it just takes the 30 or however many seconds importing 30,000 beneficiaries takes.  We don't import that many locally so I am not 100% certain how long it takes.